README.md

```diff
@@ -27,8 +27,30 @@ Usage
 require 'resque'
 require 'resque/throttler'
 
-# Rate limit at 10 jobs from `my_queue` per minute
-Resque.rate_limit(:my_queue, :at => 10, :per => 60)
+# Rate limit at 10 jobs from `my_queue` per minute, ignores jobs that have taken more than an hour
+Resque.rate_limit(:my_queue, at: 10, per: 60, max_duration: 3600)
```

Comment:
Would `timeout` or `worker_timeout` be a better option over `max_duration` here. Job that are >= the timeout are considered complete and do not count toward the limit.

Reply:
`job_timeout` seems reasonable. I'll switch to that.